edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/Program.cs

```diff
@@ -24,6 +25,7 @@ namespace Microsoft.Azure.Devices.Edge.Hub.Service
     public class Program
     {
         const int DefaultShutdownWaitPeriod = 60;
+        const SslProtocols DefaultSslProtocols = SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12;
```

Comment:
>DefaultSslProtocols [](start = 27, length = 19)

AMQP library has SslProtocols.Default by default
HTTP protocol head the current default is Tls11 and Tls12, that means we open to support Tls1.0 by default which is less secure.
MQTT has SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12 by default

Reply:
Confirmed with Anca - AMQP library we use also only supports SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12. So we should be good from a backward compat perspective.